tests/Fixture/config/multi_transport.yaml

```diff
@@ -4,8 +4,14 @@ imports:
 framework:
     messenger:
         transports:
-            async1: test://
-            async2: test://?intercept=false&catch_exceptions=false
+            async1:
+                dsn: test://
+                retry_strategy:
+                    max_retries: 0
```

Comment:
What is the purpose of having the `retry_strategy `set here if we aren't testing it? Should we be explicitly testing it?

Reply:
Actually the default retry strategy is: 
```php
new MultiplierRetryStrategy(
    maxRetries: 3,
    delayMilliseconds: 1000,
    multiplier: 1,
    maxDelayMilliseconds: 0
);
```

but it wasn't working before this PR because the `$retryStrategyLocator` in `SendFailedMessageForRetryListener` was failing to find it. Now it works, then the default retry strategy is used and the test which are testing failures dispatch more messages than expected.

Reply:
Got it.